.konflux/bundle/bundle.konflux.Dockerfile

```diff
@@ -0,0 +1,86 @@
+# Runtime image is used to overlay clusterserviceversion.yaml for Konflux
```

Comment:
regarding the dockerfile name, why to repeat the konflux.bunlde words if the parent directories state so already?

Reply:
I used the convention we used in other projects for consistency .  no special reason.

Reply:
I see. I left a comment on that for others, I think it is redundant duplication especially if konflux components don't require that

Reply:
I will not change that currently on the pr since we have this configured already in konflux tenant